src/Confluent.SchemaRegistry.Serdes.Json/JsonSerializerConfig.cs

```diff
@@ -64,13 +64,21 @@ public static class PropertyNames
             /// <summary>
             ///     Specifies whether or not the JSON serializer should use the latest subject
             ///     version for serialization.
-            ///     WARNING: There is no check that the latest schema is backwards compatible
-            ///     with the schema of the object being serialized.
             ///
             ///     default: false
             /// </summary>
             public const string UseLatestVersion = "json.serializer.use.latest.version";
 
+            /// <summary>
+            ///     Specifies whether or not the JSON serializer should check the backwards compatibility 
+            ///     with the latest schema of the subject.
+            ///     WARNING: There is no check that the latest schema is backwards compatible
+            ///     with the schema of the object being serialized when default.
+            ///
+            ///     default: false
+            /// </summary>
+            public const string LatestCompatStrict = "json.serializer.latest.compat.strict";
```

Comment:
why truncate compatibility? in java, the config property is `latest.compatibility.strict`.

`public const string LatestCompatibilityStrict = "json.serializer.latest.compatibility.strict";`